src/stories/modal-details/modal-details.scss

```diff
@@ -0,0 +1,104 @@
+$MODAL_DETAILS_CONTAINER_WIDTH: 1000px;
+
+.modal-details {
+  flex-direction: column;
+  align-items: center;
+}
+
+.modal-show {
+  display: flex;
+  opacity: 1;
+}
+
+.modal-details__container {
+  margin: 0 24px;
+  max-width: $MODAL_DETAILS_CONTAINER_WIDTH;
+
+  @include breakpoint-s {
+    margin: 0 45px;
+  }
+}
+
+.modal-details__header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @include breakpoint-s {
+    flex-direction: row;
+    align-items: center;
+    padding-top: 96px;
+    width: 80%;
+  }
+}
+
+.modal-details__cover {
+  margin: 48px 0;
+  @include breakpoint-s {
+    margin: 0;
+  }
+}
+
+.modal-details__material {
+  margin: 0;
+
+  @include breakpoint-s {
+    margin: 0 46px;
+  }
+}
+
+.modal-details__tags {
+  display: flex;
+  grid-column-gap: 8px;
+  margin-bottom: 16px;
+}
+
+/* 
+  We disable stylelint here since -webkit-box 
+  uses ellipsis for truncating which we need.
+  https://caniuse.com/?search=line-clamp
+*/
+
+/* stylelint-disable value-no-vendor-prefix */
+.modal-details__title {
+  display: block;
+  display: -webkit-box;
```

Comment:
@PELN: This effectively disables the `value-no-vendor-prefix` rule for the rest of the file. I do not think that was what you intended :smile:. [Please use `stylelint-disable-next-line` instead](https://stylelint.io/user-guide/ignore-code/).

Reply:
I have changed it now :)